VAMobile/src/screens/auth/LoginScreen/LoginScreen.tsx

```diff
@@ -45,11 +49,18 @@ const LoginScreen: FC = () => {
 
   const { demoMode } = useSelector<RootState, DemoState>((state) => state.demo)
 
-  const onFacilityLocator = navigateTo('Webview', {
-    url: WEBVIEW_URL_FACILITY_LOCATOR,
-    displayTitle: t('common:webview.vagov'),
-    loadingMessage: t('common:webview.valocation.loading'),
-  })
+  const onFacilityLocator = () => {
+    logAnalyticsEvent(Events.vama_find_location())
+    navigation.navigate(
+      'Webview' as never,
```

Comment:
Haven't seen this `as never` before.  How come it isn't needed in the simiular Webview launch in `HomeScreen.tsx` above?

Reply:
This has to have been an autocomplete unintended line. That's my guess

Reply:
To fix this without using `never`, you can update the assignment of `navigation` to use the `StackNavigationProp`, like:

```typescript
 const navigation = useNavigation<StackNavigationProp<RootNavStackParamList, keyof RootNavStackParamList>>()
```

This should also type check the name of the screen that's entered in the `navigate` function

Reply:
To fix this without using `never`, you can update the assignment of `navigation` to use the `StackNavigationProp` type, like:

```typescript
 const navigation = useNavigation<StackNavigationProp<RootNavStackParamList, keyof RootNavStackParamList>>()
```

This should also allow for type check the name of the screen that's entered in the `navigate` function

Reply:
updated